src/styles/main.scss

```diff
@@ -0,0 +1,3 @@
+@import 'utilis/variables';
+@import 'utilis/base';
+@import 'blocks/calendar';
```

Comment:
The import statement for `blocks/calendar.scss` is correct, assuming the file exists and is in the correct directory. Ensure that the path is correct and the file is present.